pkg/logs/metrics/metrics.go

```diff
@@ -0,0 +1,39 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2018 Datadog, Inc.
+
+package metrics
+
+import (
+	"expvar"
+	"strings"
+
+	"github.com/DataDog/datadog-agent/pkg/logs/status"
+)
+
+var (
+	logsExpvars    *expvar.Map
+	agentWarnings  = expvar.String{}
+	agentIsRunning = expvar.Int{}
+	// LogsProcessed is the total number of processed logs.
+	LogsProcessed = expvar.Int{}
+	// LogsSent is the total number of sent logs.
+	LogsSent = expvar.Int{}
+	// DestinationErrors is the total number of network errors.
+	DestinationErrors = expvar.Int{}
+	// TODO: Would be nice to also have these per-destination.
```

Comment:
I'd remove this `TODO`, as we don't plan to provide more than "Best Effort" support multi destinations for now

Reply:
Good point, removing this